tests/providers/apache/hdfs/hooks/test_webhdfs.py

```diff
@@ -67,7 +68,7 @@ def test_get_conn_kerberos_security_mode(
         conn = self.webhdfs_hook.get_conn()
 
         connection = mock_get_connections.return_value[0]
-        mock_kerberos_client.assert_called_once_with(f'http://{connection.host}:{connection.port}')
+        mock_kerberos_client.assert_called_once_with(f'http://{connection.host}:{connection.port}', session=None)
```

Comment:
Could you please add test cases with "use_ssl" and `verify`? (Maybe a @parameterized) test even

Reply:
Added some tests